app/assets/tailwind/components/_public-theme.css

```
@@ -0,0 +1,31 @@
+  --public-ink-muted: #58789b;
+  --public-line: #7894b2;
+  --color-text-primary: var(--public-ink);
+  --color-text-secondary: var(--public-ink-muted);
+  --color-long-text-heading: var(--public-ink);
+  --color-long-text-text: var(--public-ink);
+  --color-long-text-muted: var(--public-ink-muted);
+  --color-long-text-border: var(--public-line);
+  --color-long-text-quote-border: var(--public-line);
+  --font-family-base: "Noto Sans JP", sans-serif;
+  --font-family-mono: "Noto Sans JP", sans-serif;
+  background: #f7f5f0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)